src/components/About/About.test.js

```diff
@@ -1,9 +1,19 @@
 import React from "react";
 import About from "./AboutContainer";
+import {
+  shallow
+} from "enzyme";
+
+let wrapper;
 
 //Need at least 1 test to pass Travis CI
-describe ("<AboutContainer />", () => {
-  it ("creates component without crashing", () => {
-      const about = <About/>;
-    });
+describe("<AboutContainer />", () => {
+
+  beforeEach(() => {
+    wrapper = shallow( < About / > )
+  })
```

Comment:
Please remove beforeEach.

Please modify to `const about = shallow( < About/> )` and move down to just above the expectation on line 17.
You'll need to change `wrapper` to `about`.